src/BizHawk.Emulation.Cores/Consoles/Nintendo/BSNES/BsnesCore.ISettable.cs

```diff
@@ -29,25 +44,15 @@ void SNES.IBSNESForGfxDebugger.PutSettings(SNES.IBSNESForGfxDebugger.SettingsObj
 
 		public PutSettingsDirtyBits PutSyncSettings(SnesSyncSettings o)
 		{
-			bool ret = o.LeftPort != _syncSettings.LeftPort
-				|| o.RightPort != _syncSettings.RightPort
-				|| o.LimitAnalogChangeSensitivity != _syncSettings.LimitAnalogChangeSensitivity
-				|| o.Entropy != _syncSettings.Entropy
-				|| o.RegionOverride != _syncSettings.RegionOverride
-				|| o.Hotfixes != _syncSettings.Hotfixes
-				|| o.FastPPU != _syncSettings.FastPPU
-				|| o.FastDSP != _syncSettings.FastDSP
-				|| o.FastCoprocessors != _syncSettings.FastCoprocessors
-				|| o.UseSGB2 != _syncSettings.UseSGB2;
```

Comment:
Oh yeah, and like the settings dialog refactor, can you do this separately? Actually, maybe we should do this for all cores?